src/grammar/raise.rs

```diff
@@ -0,0 +1,40 @@
+use crate::Parser;
+use source_gen::lexer::TokenKind;
+use source_gen::syntax::SyntaxKind;
+use source_gen::T;
+
+use super::parse_ident;
+
+pub fn parse_raise_stmt(p: &mut Parser) {
+    p.start(SyntaxKind::RaiseStmt);
+    p.expect(T![raise]);
+    parse_ident(p, 1..2);
```

Comment:
The oracle examples only showed locally defined exceptions (for example within the `DECLARE` of the procedure), or internal exceptions, neither of which are schema-qualified.
According to the railroad diagram in https://docs.oracle.com/en/database/oracle/oracle-database/19/lnpls/RAISE-statement.html, the exception is also optional.

Reply:
Like I said, exceptions are optional. 
![image](https://github.com/user-attachments/assets/82de4ad6-1615-47c3-b080-f5ddf6890fee)